ingestion/src/metadata/ingestion/source/pipeline/fivetran/client.py

```diff
@@ -86,3 +88,14 @@ def get_connector_schema_details(self, connector_id: str) -> dict:
         """
         response = self.client.get(f"/connectors/{connector_id}/schemas")
         return response["data"]["schemas"]
+
+    def get_connector_column_lineage(
+        self, connector_id: str, schema_name: str, table_name: str
+    ) -> dict:
+        """
+        Method returns destination details
+        """
+        response: Optional[Response] = self.client.get(
+            f"/connectors/{connector_id}/schemas/{schema_name}/tables/{table_name}/columns"
+        )
+        return response["data"]["columns"]
```

Comment:
Are these keys guaranteed to be there from the response?

Reply:
I will take a look, AFAIK, it sent an empty array within columns key, I will validate again with the scenario.

Reply:
I have handled the fallback here, it can either be a columns array or empty array